CampaignConfig/mdc2020_beam.cfg

```diff
@@ -733,15 +741,22 @@ global.stage_name = MuStopPileup
 job_output.dest = %(outdir_fcl)s/%(stage_name)s/%(desc)s/fcl/
 
 job_setup.prescript = samweb list-file-locations --defname=sim.%(submitter)s.MuminusStopsCat.%(desc)s.art --schema=root | cut -f1 > inputs.txt
+job_setup.prescript_2 = dataset=sim.%(submitter)s.MuminusStopsCat.%(desc)s.art
+job_setup.prescript_3 = nfiles=`/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/%(release)s/link/Production/MDC2020/POMS/samCountFiles.sh $dataset`
+job_setup.prescript_4 = nevts=`/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/%(release)s/link/Production/MDC2020/POMS/samCountEvents.sh $dataset`
+job_setup.prescript_5 = let nskip=nevts/nfiles
+job_setup.prescript_6 = echo '#include "Production/JobConfig/pileup/MuStopPileup.fcl"' >> template.fcl
+job_setup.prescript_7 = echo 'physics.filters.TargetStopResampler.mu2e.MaxEventsToSkip: '${nskip} >> template.fcl
+job_setup.prescript_8 = let njobs=%(njobs)s/10
 
 executable.name = true
 executable_2.name = generate_fcl
 executable_2.arg_4 = --events-per-job=400000
-executable_2.arg_5 = --njobs=50
+executable_2.arg_5 = --njobs=$njobs
```

Comment:
This seems to be setting all the jobs (mustop resampler, neutrals resampler, ...) to the same number of jobs.  We want those in the ratio we discussed before.  Or did I miss that the njobs variable is being overwritten?

Reply:
This `$njobs` corresponds to the variable defined just above `job_setup.prescript = let njobs=%(njobs)s/100` which where `%(njobs)s` is the number of POT jobs. It's probably confusing to use the same name for INI and bash variable... But I think is correct right?

Reply:
I was just looking at the snippets of changed lines, so it's missing some context.  As long as the ratios of the numbers of jobs ends up as we discussed it's fine.  Can you run a test to verify what ratios are produced?

Reply:
In order to run a test I need a tagged version because scripts like `samCountFiles.sh` must be available on cvmfs. For example I do:
```
job_setup.prescript_3 = nfiles=`/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/%(release)s/link/Production/MDC2020/POMS/samCountFiles.sh $dataset`
```